apps/webpack.js

```diff
@@ -150,7 +149,6 @@ var baseConfig = {
   },
   module: {
     rules: [
-      {test: /\.exported_json$/, type: 'asset/source'},
```

Comment:
out of curiosity, how did you know that `.exported_json` and `.exported_js` were expo-specific?

Reply:
haha good question -- before I started deleting, I skimmed through the list of PRs you shared (thanks for sharing that btw!) to get a sense of the feature in general, and took some notes on places where I saw things were added. [Believe this config was added here.](https://github.com/code-dot-org/code-dot-org/pull/21687)

Reply:
ahhh, perfect, thank you!